README.org

```diff
@@ -3,23 +3,33 @@
   A Clojure library designed to parse mode files into clojure data
   structures.
 
+** Status
+   Usable, but not perfect, parse the [[test/orgmode/test.org][test]] file to see some of the issues.
+
 ** Installation
 
-   If you are using =lein=, then just add it as a dependency.
+   Using =deps.edn=
 
 #+BEGIN_SRC clojure
-  [bnbeckwith.com/orgmode "0.7.2"]
+  {:deps {orgmode {:git/url "https://github.com/davidpham87/orgmode"
```

Comment:
Should this be a link to my repository?

Reply:
Yes it should be your repository, once we get the merge.